example/pico-hello-world/README.md

```diff
@@ -9,9 +9,10 @@ cmake ..
 make
 ```
 
-This should produce a `firmware.uf2` file to flash onto the pico-ice RP2040 chip.
+This should produce a `pico_hello_world.uf2` file in the build directory to flash onto the pico-ice RP2040 chip.
 See the [main `README.md`](../../README.md) for how to do this.
 
 Then it will print "hello world" in loop across the USB-UART #0 (`firmware.c`)
+BaudRate = 115200
```

Comment:
I think that with USB CDC you can set any baudrate at all, it goes at USB pace either way.

If using plain UART, maybe that is another question.